src/components/CustomerQuota/NoQuota/NoQuotaCard.tsx

```diff
@@ -154,6 +154,12 @@ export const sortTransactions = (
     });
 };
 
+const getLatestTransactionTime = (quotaResponse: Quota) =>
```

Comment:
There is still a slight delay in loading latest transaction time in the title after making this change, so I am waiting for Pat and Eida to feedback before writing the unit test for this~

Reply:
added test le!!